envhelp/makeconfig.py

```diff
@@ -0,0 +1,92 @@
+# -*- coding: utf-8 -*-
+#
+# --- BEGIN_HEADER ---
+#
+# test_mig_shared_compat - unit test of the corresponding mig shared module
```

Comment:
**pedantic**: this header line always needs adjustment to match module.

Reply:
copy pasta